src/mpid/ch4/shm/posix/posix_coll_init.c

```diff
@@ -19,7 +19,7 @@
     - name        : MPIR_CVAR_BCAST_POSIX_INTRA_ALGORITHM
       category    : COLLECTIVE
       type        : string
-      default     : auto
+      default     : release_gather
```

Comment:
I assume this is a HACK commit and will be removed from the final PR.

Reply:
No. I think shared memory collectives should be enabled by default after the intro node topology aware trees are merged. So this patch should actually be merged.